ios/CleverTapReact/CleverTapReact.mm

```diff
@@ -264,6 +264,43 @@ - (void)setCleverTapInstance:(CleverTap *)instance {
     [self returnResult:result withCallback:callback andError:nil];
 }
 
+RCT_EXPORT_METHOD(getUserEventLog:(NSString*)eventName callback:(RCTResponseSenderBlock)callback) {
+    RCTLogInfo(@"[CleverTap getUserEventLog: %@]", eventName);
+    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0), ^{
```

Comment:
Can we use priority as default here and other places as background priority is used for cases where heavy operation is needed, we are fetching data only here. - DISPATCH_QUEUE_PRIORITY_DEFAULT